src/aws/osml/tile_server/main.py

```diff
@@ -87,16 +85,38 @@ async def lifespan(app: FastAPI) -> AbstractAsyncContextManager[None] | FastAPI:
         They represent your AWS S3 bucket instance,
         your database instance and a queue of requests respectively.
     """
-    viewpoint_worker = ViewpointWorker(viewpoint_request_queue, aws_s3, viewpoint_database, logger)
+    # startup functions before serving requests
+
+    # configure_logging()
```

Comment:
Either uncomment it or remove this line.

Reply:
updating in new commit